cpp/src/gandiva/precompiled/arithmetic_ops.cc

```diff
@@ -208,6 +208,22 @@ NUMERIC_FUNCTION(DIVIDE)
 
 #undef DIVIDE
 
+#define POSITIVE(TYPE) \
+  FORCE_INLINE         \
+  gdv_##TYPE positive_##TYPE(gdv_##TYPE in) { return static_cast<gdv_##TYPE>(in); }
```

Comment:
Since you are casting to the same type, I think you can just return the input number, right?